schemas/styles/csl-choose.rnc

```diff
@@ -64,6 +64,38 @@ div {
       attribute variable {
         list { variables+ }
       }
+      |
+      condition.atts.complex.dates
+      |
+      condition.atts.complex.general
+
+  
+  # Tests that require two attributes 
+  # Dates
+  condition.atts.complex.dates =
+    ## specifies the variable to be tested
+    attribute tested { 
+      list { variables.dates+ }
+     },
+     (date-precision | date-range)+
+    
+  ## Tests the precision of a date variable
+  date-precision = attribute date-precision { text }
```

Comment:
why is the value text?

Reply:
because I still have to add the list... ;-) Getting multiple conditions to work got in the way.

Reply:
Should be ok now.

Reply:
Status of this comment?

Reply:
Should be ok now. (Just one question: Does it need to be the a list if you can only select one of the values?)

Reply:
> Does it need to be the a list if you can only select one of the values?)

No; list refers to the value.